apache/trafficcontrol
# Enable DNSSEC for CDN in a Box

## What does this PR (Pull Request) do?

- - [x] This PR fixes #5607 by, if DNSSEC is enabled, attempting to generate DNSSEC keys until they are generated, and only enrolling Delivery Services after that point.
- Enables DNSSEC for CDN in a Box by default

## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Traffic Router integration tests

## What is the best way to verify this PR?

1. Verify that CDN in a Box starts successfully now that DNSSEC is enabled
2. Run the Traffic Router integration tests

## If this is a bug fix, what versions of Traffic Control are affected?

- master (038a439ebe)

## The following criteria are ALL met by this PR
